apps/web/screens/IcelandicGovernmentInstitutionVacancies/IcelandicGovernmentInstitutionVacanciesList.tsx

```diff
@@ -380,294 +489,262 @@ const IcelandicGovernmentInstitutionVacanciesList: Screen<
-                                    variant="purple"
-                                  >
-                                    {n(
-                                      'applicationDeadlineTo',
-                                      'Umsóknarfrestur',
-                                    )}{' '}
-                                    {vacancy.applicationDeadlineTo}
-                                  </Tag>
-                                </Box>
-                              )}
-                            </Stack>
-                          </GridColumn>
-
-                          <GridColumn>
-                            <Box width="full">
-                              {logoUrl && (
-                                <>
-                                  <Hidden below="lg">
-                                    <img
-                                      className={styles.logo}
-                                      src={logoUrl}
-                                      alt=""
-                                    />
-                                  </Hidden>
-                                  <Hidden above="md">
-                                    <Box
-                                      display="flex"
-                                      justifyContent="center"
-                                      width="full"
-                                    >
-                                      <img
-                                        className={styles.logo}
-                                        src={logoUrl}
-                                        alt=""
-                                      />
-                                    </Box>
-                                  </Hidden>
-                                </>
-                              )}
-                            </Box>
-                          </GridColumn>
-                        </GridRow>
-                      </Box>
-                    </FocusableBox>
-                  </GridColumn>
-                )
-              })}
-          </GridRow>
-          {filteredVacancies.length > 0 && (
-            <Box paddingTop={8}>
-              <Pagination
-                variant="blue"
-                page={selectedPage}
-                itemsPerPage={ITEMS_PER_PAGE}
-                totalItems={filteredVacancies.length}
-                totalPages={totalPages}
-                renderLink={(page, className, children) => (
-                  <button
-                    onClick={() => {
-                      setSelectedPage(page)
-                    }}
-                  >
-                    <span className={className}>{children}</span>
-                  </button>
+          </SidebarLayout>
+        )}
+        {isTablet && (
+          <Box marginX={3} paddingTop={3}>
+            <Stack space={1}>
+              <Text variant="h5" as="h4" paddingY={1}>
+                {n('search', 'Leit')}
+              </Text>
+              <FilterInput
```

Comment:
I suggest for the mobile input that we have an onkeydown event that removes the focus on the input when enter is pressed.

This will cause the on screen keyboard on phones to go away (it also was this way previously)